can-control_test.js

```diff
@@ -290,6 +290,19 @@ test("drag and drop events", function() {
 	domDispatch.call(draggable, "dragend");
 });
 
+test("beforeremove event", function() {
+  expect(1);
+  var Foo = Control.extend("", {
+    "beforeremove": function() {
+      ok(true, "beforeremove called");
+    }
+  });
+  this.fixture.appendChild(fragment('<div id="foo"/>'));
```

Comment:
Since you're dispatching the `beforeremove` event yourself, you probably don't need to actually append your element to the DOM. Not using the DOM will make the test run faster.

Something like this should test the same thing:
```
  var el = fragment('<div id="foo"/>');
  new Foo(el);
  domDispatch.call(el, "beforeremove");
```